README.md

```diff
@@ -0,0 +1,6 @@
+# [Stackdriver Monitoring Agent](https://cloud.google.com/monitoring/agent/)
+
+This is Stackdriver Monitoring agent's implementation. It is forked from collectd ([official website](https://collectd.org/), [repo](https://github.com/collectd/collectd)).
+
+You can still access original README of collectd at
+[README_COLLECTD](README_COLLECTD).
```

Comment:
[Optional] Instead of hosting a copy locally called `README_COLLECT`, we could also just point to:
https://github.com/collectd/collectd/blob/collectd-5.5.2/README